It is not upstream business to decide what build style we can use specially when they don't cover all the accpetable values from CMake itself.

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f3e69b8..5c111ea 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -15,13 +15,6 @@ endif()
 set(Eigen_SOURCE_DIR ${Eigen3_SOURCE_DIR})
 set(Eigen_BINARY_DIR ${Eigen3_BINARY_DIR})
 
-# guard against bad build-type strings
-
-if (NOT CMAKE_BUILD_TYPE)
-  set(CMAKE_BUILD_TYPE "Release")
-endif()
-
-
 #############################################################################
 # retrieve version information                                               #
 #############################################################################
